### DISCUSSION TITLE
I tried so many way but I just can't get verified for github education.

### DISCUSSION THREAD

**Discussion by 12356007** [community/community#53596] [GitHub Education]
I had tried school ID( but I know my school id don't have date).
I also tried screenshot of school portal, official transcript, class schedule for the semester. All failed !!?!?!?
I event applied the prove of enrollment from school.
Is it the language problem that I can't get verified?
All the prove is in Chinese.
Plz help me

**Comment by samus-aran** [comment]
Hi @12356007  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Comment by whitewatercn** [comment]
可以参考我的这个帖子再逝一逝 https://xlog.whitewater.wang/Github_Student_Developer_Pack
You can refer to this post to try again https://xlog.whitewater.wang/Github_Student_Developer_Pack
